### PR TITLE
@damassi => [Collect] Re-add the color wheel filter

### PR DESCRIPTION
**Pull request by mzikherman**
screen shot 2019-01-23 at 5 35 51 pm

Requires a palette bump

### CODE REVIEW

File: src/Apps/Collect/Components/Filters/ColorFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
+import { CheckIcon } from "@artsy/palette"
+import { FilterState } from "Apps/Collect/FilterState"
+import { ContextConsumer } from "Artsy/SystemContext"
+import React from "react"
+
+interface Props {
+  filters: FilterState
+}
+
+const CheckmarkPositions = {
```
Comment (damassi) [inline src/Apps/Collect/Components/Filters/ColorFilter.tsx]: this is nitpicky -- but if this is just an object

`const checkmarkPositions =` 

CapsCase is idomatic for classes and components, but not for anything else.

File: src/Apps/Collect/Components/Filters/ColorFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    }
+  }
+
+  render() {
+    const { filters } = this.props
+
+    return (
+      <ContextConsumer>
+        {({ mediator }) => {
+          const CheckmarkStyle = {
```
Comment (damassi) [inline src/Apps/Collect/Components/Filters/ColorFilter.tsx]: same here

File: src/Apps/Collect/Components/Filters/ColorFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+            position: "relative",
+            ...CheckmarkPositions[filters.state.color],
+          }
+          return (
+            <>
+              <svg
+                version="1.1"
+                id="Layer_1"
```
Comment (damassi) [inline src/Apps/Collect/Components/Filters/ColorFilter.tsx]: these two props can be removed as they don't do anything

File: src/Apps/Collect/Components/Filters/ColorFilter.tsx [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+    return (
+      <ContextConsumer>
+        {({ mediator }) => {
+          const CheckmarkStyle = {
+            position: "relative",
+            ...CheckmarkPositions[filters.state.color],
+          }
+          return (
+            <>
```
Comment (damassi) [inline src/Apps/Collect/Components/Filters/ColorFilter.tsx]: Can simplify all of the svg props down below a bit by turning this fragment into something like 

```js
const Container = styled.div`
  cursor: pointer;
`
```

and then remove all of the individual `style = {{ cursor: 'pointer' }}` from each node.

File: src/Apps/Collect/Components/Filters/ColorFilter.tsx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+                x="0px"
+                y="0px"
+                viewBox="0 0 175 175"
+                enableBackground="new 0 0 175 175"
+                xmlSpace="preserve"
+              >
+                <g>
+                  <path
+                    fill="#F7923A"
+                    style={{ cursor: "pointer" }}
```
Comment (damassi) [inline src/Apps/Collect/Components/Filters/ColorFilter.tsx]: these can be removed by adding a container higher up that makes the whole thing a pointer

Comment (mzikherman) [reply src/Apps/Collect/Components/Filters/ColorFilter.tsx]: Actually I only want the wedges to be clickable. There's other parts (like the dividing lines) that shouldn't be pointer. That's why I added this explicitly to just the wedges.

Comment (damassi) [reply src/Apps/Collect/Components/Filters/ColorFilter.tsx]: Aren't the dividing lines like really narrow tho? I don't think a user would notice

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a3b12d8851dbc2a1c1352832dbe284e6ddca5418

https://deploy-preview-1932--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Will merge and follow up with a bit of cleanup :)

**Pull request merged**